Makefile

```diff
@@ -12,6 +12,10 @@ NODE_MODULES_TEST=package-lock.json
 # one example file in dist/ from webpack to check if that already ran
 WEBPACK_TEST=dist/index.html
 
+WEBLATE_REPO=cockpit-podman-weblate
```

Comment:
Just in case we want to port that to starter-kit, could we generalize this a bit? This is just a temporary directory name, so it doesn't have to contain the project name. Also, would be nice to put this into ./tmp/ or so, and add that to .gitignore, to further point out that it can be cleaned up. It's not currently removed automatically, as it's being used by two different rules. But it should at least be removed in `make clean`.